packages/react-frontend/src/index.css

```diff
@@ -56,3 +56,83 @@ h2 {
   font-size: 240px; 
   font-weight: bold;
 }
+
+footer {
+  background-color: #690606;
+  padding-top: 6px; /* Only top padding */
+  width: 100vw;
+  box-sizing: content-box; /* Ensures padding is included in the width */
+  margin-top: 10%; /* Pushes footer to the bottom */
+  padding-bottom: 10px; /* Ensure no padding at the bottom */
+  left: 0;
+  right: 0;
+  position: relative;
+}
+
+.footer-container {
+  display: flex;
+  justify-content: space-around;
+  max-width: 1200px;
+  margin: 0 auto;
+  width: 100%;
+  box-sizing: border-box; /* Ensures padding is included in the width */
+}
+
+.footer-column {
+  flex: 1;
+  text-align: left;
+}
+
+.footer-column h3 {
+  margin-bottom: 10px;
+}
+
+.footer-column a {
+  display: block;
+  color: #dfbebe;
+  text-decoration: none;
+  margin-bottom: 5px;
+  font-family: Inter, system-ui, Avenir, Helvetica, Arial, sans-serif;
+  font-size: 18px;
+  font-weight: 400;
+  
+}
+
+.footer-column a:hover {
+  text-decoration:color-mix(in hsl shorter hue, color percentage, color percentage);
```

Comment:
was this line meant to be a comment instead? or are 'color percentage' and 'in hsl shorter hue' valid parameters?